packer/http/centos-7.2/ks-ganeti.cfg

```diff
@@ -0,0 +1,90 @@
+install
+cdrom
+lang en_US.UTF-8
+keyboard us
+network --bootproto=dhcp
+rootpw osuadmin
+firewall --disabled
+selinux --permissive
+timezone UTC
+unsupported_hardware
+bootloader --location=mbr
+text
+skipx
+zerombr
+clearpart --all --initlabel
+part /boot --fstype="ext4" --size=256
+part swap --fstype="swap" --size=512
+part / --fstype="ext4" --grow --size=100
+auth --enableshadow --passalgo=sha512 --kickstart
+firstboot --disabled
+reboot
+services --disabled=NetworkManager
+user --name=osuadmin --plaintext --password osuadmin
+url --url=http://centos.osuosl.org/7/os/x86_64/
+repo --name=updates --baseurl=http://centos.osuosl.org/7/updates/x86_64/
```

Comment:
Let's go ahead and statically set this to ``7.2`` so it always builds that version.